src/microfrontends/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@harness/microfrontends",
-  "version": "1.63.0",
+  "version": "1.63.1",
```

Looks like this is not a bug fix ticket, so the version should be minor not patch bumped
```suggestion
  "version": "1.64.0",
```

---

done